scripts/sanity/sanity.py

```diff
@@ -0,0 +1,20 @@
+import os, shutil
+
+####################### Sanity Steps ###########################
+def installGitHooks():
+    if os.path.exists("./git-hooks/pre-commit"):
+        print("Git hooks step -- Copying pre-commit")
+        shutil.copy2("./git-hooks/pre-commit", "../../.git/hooks/pre-commit")
+
+    if os.path.exists("./git-hooks/pre-push"):
+        print("Git hooks step -- Copying pre-push")
+        shutil.copy2("./git-hooks/pre-push", "../../.git/hooks/pre-push")
+
+########################### Main ###############################
+def main():
+    print("Running sanity checks")
```

Comment:
Considering that this script is only copying the hooks and not running them, shouldn't this message display something like "Hooking sanity checks to GIT..."?

Reply:
Done!